app/services/idv/profile_maker.rb

```diff
@@ -48,7 +48,11 @@ def save_profile(
 
     def set_idv_level(in_person_verification_needed:, selfie_check_performed:)
       if in_person_verification_needed
-        :legacy_in_person
+        if IdentityConfig.store.in_person_proofing_enforce_tmx
```

Comment:
Noting here that I will add a check for `FeatureManagement.proofing_device_profiling_decisioning_enabled?` in a separate pull request.